### ISSUE TITLE
invoice object returns a Vec<Parcel> as type "parcel"

### ISSUE THREAD

**Issue by bacongobbler** [deislabs/bindle#333]
Should these fields be made plural to match other fields?

https://github.com/deislabs/bindle/blob/9487388a7e306c10a9c71bd6adac1aafde1678be/src/invoice/mod.rs#L79-L81

IMO we should be listing these as "parcels", "groups", and "signatures" as we're dealing with a list of objects rather than a single object.

That being said, this would break backwards compatibility with older bindle clients.

I noticed this while updating [bindle-dotnet](https://github.com/deislabs/bindle-dotnet) to the newest bindle APIs.

**Comment by thomastaylor312** [comment]
We had it that way originally but changed our minds for a specific reason I can't remember. @technosophos do you remember?

**Comment by itowlson** [comment]
#34

**Comment by thomastaylor312** [comment]
Yeah that is where we changed it, but we had an "in person" discussion about why we were doing it
